pkg/sql/sqlbase/system.go

```diff
@@ -162,8 +162,12 @@ CREATE TABLE system.jobs (
 	created           TIMESTAMP NOT NULL DEFAULT now(),
 	payload           BYTES     NOT NULL,
 	progress          BYTES,
+	created_by_type   STRING,
+	created_by_id     INT,
 	INDEX (status, created),
-	FAMILY (id, status, created, payload),
+	INDEX (created_by_type, created_by_id) STORING (status),
+
+	FAMILY "primary" (id, status, created, payload, created_by_type, created_by_id),
```

Comment:
huh, I _guess_ it is OK to change the name of the default family for new jobs tables but not migrate it for old ones? I donno, maybe a question for @lucy-zhang

Reply:
Oh, yeah, what is preventing us from using the existing name?